9.mAP/notebooks/0.generate_map_scores_class_level.ipynb

```diff
@@ -0,0 +1,954 @@
+{
```

Comment:
<p>Something looks off about the p-values. It's alternating between <code>0.5</code> and <code>1,0</code> Considering revising the generated AP scores and see if that pattern is reoccurring. If you have an explanation of this, please feel free to disregard this comment.  </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>